dm/ctl/master/operate_source.go

```diff
@@ -26,7 +26,7 @@ import (
 // NewOperateSourceCmd creates a OperateSource command
 func NewOperateSourceCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "operate-source <operate-type> <config-file>",
+		Use:   "operate-source <operate-type> <config-file> [config-file ...]",
 		Short: "create/update/stop upstream MySQL/MariaDB source",
```

Comment:
Maybe `operate-source <operate-type> <config-file> [config-file ...]`

Reply:
want to express "at lease one `config-file`"